opus_codec_registration.src.html

```diff
@@ -131,16 +131,26 @@
 <xmp>
 dictionary OpusEncoderConfig {
   OpusBitstreamFormat format = "opus";
+  [EnforceRange] unsigned long frameDuration;
 };
 </xmp>
 </pre>
 
+To check if an {{OpusEncoderConfig}} is valid, run these steps:
+1. If {{OpusEncoderConfig/frameDuration}} is not a valid ptime value,
```

Comment:
NIT: should ptime be `ptime`? (wrapped with backticks?)

Reply:
Yes, it should be.